app/Metadata/Extractor.php

```diff
@@ -182,60 +183,186 @@ public function extract(string $filename, string $kind): array
-						// because the takestamp is actually in UTC.  This will
-						// trigger, e.g., for mp4 files with the Exiftool
-						// extractor.  We recreate the takestamp as a UTC
-						// timestamp and _then_ change the time zone to local.
-						$takestamp = new \DateTime($takestamp->format('Y-m-d H:i:s'), new \DateTimeZone('UTC'));
-						$takestamp->setTimezone(new \DateTimeZone(date_default_timezone_get()));
-					} elseif ($takestamp->getTimezone()->getName() === 'Z') {
-						// This one is correctly in Zulu (UTC).  We just need
-						// to change the time zone to local.
-						$takestamp->setTimezone(new \DateTimeZone(date_default_timezone_get()));
-					} else {
-						// In the remaining case the time zone information was
-						// extracted and the takestamp is assumed to be in local
-						// time, so we don't need to do anything.
-						//
-						// The only known example are the mov files from Apple
-						// devices; the time zone will be formatted as "+01:00"
-						// so neither of the two conditions above should trigger.
+						// because the recording time is actually in UTC.
+						// This will trigger, e.g., for mp4 files with the
+						// Exiftool extractor.
+						// We recreate the recording time as a UTC timestamp
+						// and _then_ change the timezone to the application's
+						// default timezone.
+						// Note: This assumes that the application's default
+						// timezone is the same as the timezone of the
+						// location where the video has been recorded and that
+						// the beholder (of the video) expects to observe
+						// that timezone.
+						$taken_at = new \DateTime(
+							$taken_at->format('Y-m-d H:i:s'),
+							new \DateTimeZone('UTC')
+						);
+						$taken_at->setTimezone(new \DateTimeZone(date_default_timezone_get()));
+					} elseif ($taken_at->getTimezone()->getName() === 'Z') {
+						// This one is correctly in Zulu (UTC).
+						// We change the timezone to the application's default
+						// timezone and convert the time.
+						// Note: This assumes that the application's default
+						// timezone is the same as the timezone of the
+						// location where the video has been recorded and that
+						// the beholder (of the video) expects to observe
+						// that timezone.
+						$taken_at->setTimezone(new \DateTimeZone(date_default_timezone_get()));
 					}
+					// In the remaining cases the timezone information was
+					// extracted and the recording time is assumed exhibit
+					// to original timezone of the location where the video
+					// has been recorded.
+					// So we don't need to do anything.
+					//
+					// The only known example are the mov files from Apple
+					// devices; the time zone will be formatted as "+01:00"
+					// so neither of the two conditions above should trigger.
 				}
 			}
-
-			// We need to make sure that the takestamp is between '1970-01-01 00:00:01' UTC and '2038-01-19 03:14:07' UTC.
```

Comment:
This part is required because some photos have the wrong exif data... :(
It was added to fix a bug (I don't remember which one but a `git blame` should give us this information).

Reply:
See here: https://github.com/LycheeOrg/Lychee/pull/414 and https://github.com/LycheeOrg/Lychee/issue/413

Reply:
OK, I will check and if I know the reason why it was added, I will re-add it. However, what do we do with photos which truly have been taken before 1970 or after 2038?

Reply:
I'm not sure about the after 2038. For the before 1970, I'm pretty sure we will not have this information in a digital picture. ;)

Reply:
I checked the issue, unfortunately the links to the test photos in the [post](https://github.com/LycheeOrg/Lychee/issues/413#issuecomment-568795209) are dead. I tried to create some own GIF and PNG with various values for the tags `dateTimeOrigiginal` and `xmp:dateTimeOrigiginal` but could not reproduce the error :-(

> For the before 1970, I'm pretty sure we will not have this information in a digital picture.

You may have. Some national archives digitize historic photos and set the EXIF data correctly.

Reply:
Hopefully @alex-phillips is still around. I will also try to poke him on discord. :+1:

Reply:
Thanks. It would be really great to have this photos and even put them into the test base (if @alex-phillips agrees). This way we can be sure that there is no regression. If I get a link to those photos, I will write tests for them.

Reply:
@ildyria @nagmat84 Unfortunately, I don't think I was involved in the date change here, just reported the error and opened the issue. Here is one of the problematic images I referenced in the issue if it helps.

@tmp-hallenser may have more info, but my guess is that exif field needed to be checked, but not sure why the date bounds were added.

![6a4bde8ad79732826bf1df9bd11bed4b](https://user-images.githubusercontent.com/3135216/119348827-6364b900-bc6b-11eb-8eb9-c39eaf33b153.png)

Reply:
@alex-phillips Thank you for giving me one of the problematic images. Unfortunately, everything works smoothly and I cannot reproduce the error. However, I also noticed that the EXIF data for this image do not contain a tag `dateTimeOriginal` at all. This means, the removed code path (checking for boundaries between 1970 and 2038) would have never been taken anyway and are totally unaffected by this image. @alex-phillips Are you sure that this one really has been one of the problematic images?

@ildyria How do we proceed?

Reply:
If the bug is no-longer reproducible, I would just move forward.

Reply:
Great. In that case I like to kindly ask @ildyria to also review the changes to the frontend and accept them if everything is fine. Then I can proceed with the merge.